test/verify/check-system-services

```diff
@@ -288,8 +315,22 @@ Unit=test.service
         b.wait_in_text(self.svc_sel('test-onboot.timer') + ' .service-unit-last-trigger', "unknown")  # last trigger
         self.run_systemctl(user, "stop test-onboot.timer")
 
-        # Selects Paths tab - the test VM does not have any user path units
         if not user:
+            self.goto_service("deleteme.timer")
+            self.do_action("Delete")
+            b.click("#delete-timer-modal-btn")
+            self.wait_page_load()
+            self.assertNotIn("deleteme", m.execute("systemctl list-timers"))
+            self.assertEqual(m.execute(f"! test -f {path}/deleteme.service && echo yes").strip(), "yes")
```

Comment:
Should also check that deleteme.timer does not exist any more. Also, the `yes` bit and assertEqual() shouldn't be necessary -- `m.execute()` already fails if the command fails.